contracts/programs/bubblegum/src/lib.rs

```diff
@@ -99,15 +99,15 @@ pub struct Burn<'info> {
 }
 
 #[derive(Accounts)]
-pub struct Transfer<'info> {
+pub struct TransferWithOwner<'info> {
```

Comment:
Let's get this working with the Indexer :)

Reply:
Also for the sake of not breaking existing interfaces, it might be better to do this the sketchy way for the time being

Reply:
Or just not rename the normal `Transfer` Instruction